conntrack: add page (#8081)
tldr-pages/tldr: pages/linux/conntrack.md

# conntrack

> Interact with the Netfilter connection tracking system.
> Search, list, inspect, modify, and delete connection flows.
> More information: <https://manned.org/conntrack>.

- List all currently tracked connections:

`conntrack --dump`

- Display a real-time event log of connection changes:

`conntrack --event`

- Display a real-time event log of connection changes and associated timestamps:

`conntrack --event -o timestamp`

- Display a real-time event log of connection changes for a specific IP address:

`conntrack --event --orig-src {{ip_address}}`

> - Delete all flows to the given source IP address:

Reviewer: ```suggestion
- Delete all flows to the specified source IP address:
```

Author: Applied in amended revision 1c134c63

> - Delete all flows to the specified source IP address:

Reviewer 2: ```suggestion
- Delete all flows for a specific source IP address:
```
